lib/gitlab/client/build_triggers.rb

```diff
@@ -21,7 +22,7 @@ def triggers(project)
     # @param  [Integer] project The ID of a project.
     # @param  [String] token The token of a trigger.
     # @return [Gitlab::ObjectifiedHash] The trigger.
-    def trigger(project, token)
+    def trigger_detail(project, token)
```

Comment:
I think `trigger` was a fine name for this method.  It matches the pattern used elsewhere in the gem where asking for `project` or `user` gets you details.   What do you think?